test/framework/resources/k8s/resources/daemonset.go

```diff
@@ -26,8 +27,12 @@ import (
 
 type DaemonSetManager interface {
 	GetDaemonSet(namespace string, name string) (*v1.DaemonSet, error)
+
+	CreateAndWaitTillDaemonSetIsReady(daemonSet *v1.DaemonSet, timeout time.Duration) (*v1.DaemonSet, error)
+
 	UpdateAndWaitTillDaemonSetReady(old *v1.DaemonSet, new *v1.DaemonSet) (*v1.DaemonSet, error)
 	CheckIfDaemonSetIsReady(namespace string, name string) error
+	DeleteAndWaitTillDaemonSetDeleted(daemonSet *v1.DaemonSet, timeout time.Duration) error
```

Comment:
nit:  To keep the signature similar to CreateAndWait...`DeleteAndWaitTillDaemonSetIsDeleted`

Reply:
Thanks, changed!